src/provider/HttpFile/ConfigurationServiceHttpFile.ts

```diff
@@ -22,16 +22,32 @@ import {
   getRepositoryKeyNotFoundErrorMessage
 } from '../../utils';
 
+interface ConfigurationServiceHttpFileRequest {
+  token: string,
+  disableCache?: boolean
+}
+
 export class ConfigurationServiceHttpFile implements ConfigurationService {
-  constructor(private token: string) {
+  private token: string = '';
+  private disableCache?: boolean;
+
+  constructor(request: string | ConfigurationServiceHttpFileRequest) {
```

Comment:
Please add tsdoc comment for the constructor, write in the description that passing string is deprecated and you should pass an object

Reply:
done